test/docker.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:io';
+
+import 'package:docker_process/containers/postgres.dart';
+import 'package:test/test.dart';
+
+const _kContainerName = 'postgres-dart-test';
+
+Future<void> startPostgresContainer() async {
```

Comment:
I think async happens only in the setupAll, so this method should rather be something like `void usePostgresDocker() {`.